File: cf-263a.py
matrix = [[0] * 5] * 5
for i in range(5):
    matrix[i] = input().split()

xPos = 0
yPos = 0
for i in range(5):
    for j in range(5):
        if matrix[i][j] == '1':
            xPos = i
            yPos = j

print(abs(2 - xPos) + abs(2 - yPos))

File: cf-263a.cpp
#include <iostream>
using namespace std;

int main () {
    int matrix[5][5], xPos, yPos;
    for (int i = 0; i < 5; i++)
        for (int j = 0; j < 5; j++) {
            cin >> matrix[i][j];
            if (matrix[i][j])
                xPos = i, yPos = j;
        }

    cout << abs(2 - xPos) + abs(2 - yPos);
    return 0;
}
